docs/developers/set-autoscale.md

```diff
@@ -20,46 +28,229 @@
+  
+  ```
+  $ vela port-forward helloworld 80
+  Password:
+  Forwarding from 127.0.0.1:80 -> 80
+  Forwarding from [::1]:80 -> 80
+  
+  Forward successfully! Opening browser ...
+  Handling connection for 80
+  Handling connection for 80
+  Handling connection for 80
+  Handling connection for 80
+  ```
+  
+  On your macOS, you might need to add `sudo` ahead of the command.
+  
+  Use Apache HTTP server benchmarking tool `ab` to access the application.
+  
+  ```
+  $ ab -n 10000 -c 200 http://127.0.0.1/
+  This is ApacheBench, Version 2.3 <$Revision: 1843412 $>
+  Copyright 1996 Adam Twiss, Zeus Technology Ltd, http://www.zeustech.net/
+  Licensed to The Apache Software Foundation, http://www.apache.org/
+  
+  Benchmarking 127.0.0.1 (be patient)
+  Completed 1000 requests
+  ```
+  
+  Monitor the replicas of Deployment `frontend` again, its replicas gradually increase from one to three. 
+  ```
+  $ vela status helloworld --svc frontend
+  About:
+  
+    Name:      	helloworld
+    Namespace: 	default
+    Created at:	2020-11-05 20:07:21.830118 +0800 CST
+    Updated at:	2020-11-05 20:50:42.664725 +0800 CST
+  
+  Services:
+  
+    - Name: frontend
+      Type: webservice
+      HEALTHY Ready: 1/1
       Traits:
-        - autoscale: 
-            maxReplicas=4
-            minReplicas=1
-            replicas=2
-            startAt=21:00
-            timezone=Asia/Shanghai
-            days=Monday, Tuesday
-            name=cron-test
-            type=cron
-            duration=2h
+        - ✅ autoscale: type: cpu	minReplicas: 1	maxReplicas: 5	CPUUtilization(target/current): 5%/10%	replicas: 2
+      Last Deployment:
+        Created at: 2020-11-05 20:07:23 +0800 CST
+        Updated at: 2020-11-05T20:50:42+08:00
   ```
   
-  The time is `21:07` which is in the active period of the trait which started at `21:00` and the duration is two hours.
-  Check the replicas of Deployment `frontend` again, it has been scaled to 2.
   ```
-  $ kubectl get deploy
-  NAME       READY   UP-TO-DATE   AVAILABLE   AGE
-  frontend   2/2     2            2           8m42s
+  $ vela status helloworld --svc frontend
+  About:
+  
+    Name:      	helloworld
+    Namespace: 	default
+    Created at:	2020-11-05 20:07:21.830118 +0800 CST
+    Updated at:	2020-11-05 20:50:42.664725 +0800 CST
+  
+  Services:
+  
+    - Name: frontend
+      Type: webservice
+      HEALTHY Ready: 1/1
+      Traits:
+        - ✅ autoscale: type: cpu	minReplicas: 1	maxReplicas: 5	CPUUtilization(target/current): 5%/10%	replicas: 3
+      Last Deployment:
+        Created at: 2020-11-05 20:07:23 +0800 CST
+        Updated at: 2020-11-05T20:50:42+08:00
   ```
   
-  Wait after the period ends, the replicas will be one eventually.
+  Stop `ab` tool, and the replicas will decrease to one eventually.
 
+
+docs/developers/set-autoscale.md## Scale workload by cron
+Introduce how to automatically scale workloads by cron in Appfile.
+
+### Prerequisites
```

Comment:
This section is NOT needed, developers don't know that is KEDA and we won't let them know.